OsmAnd/src/net/osmand/plus/osmedit/EditPoiDialogFragment.java

```diff
@@ -449,7 +449,8 @@ private void save() {
 			if (offlineEdit && !Algorithms.isEmpty(poiTypeTag)) {
 				node.putTagNoLC(EditPoiData.POI_TYPE_TAG, poiTypeTag);
 			}
-			comment = OsmPoint.displayAction.get(action) + " " + poiTypeTag;
+			String actionString = action == OsmPoint.Action.CREATE ? getString(R.string.shared_string_add) : getString(R.string.shared_string_edit);
```

Comment:
It should be in no_translate.xml file otherwise people will translate it. We should have only in english in default comment so we should not use shared_string... which is translatable